roles/kafka_broker/tasks/register_cluster.yml

```diff
@@ -35,3 +35,6 @@
     status_code: 204
```

Comment:
Do we still need the status_code in the uri module ?

Reply:
Thanks @nsharma-git ,
I guess yes its needed, but I'll quickly test it out

Reply:
tested, retries works